scripts/map_peaks_to_known_genes.py

```diff
@@ -16,9 +19,9 @@
 """
 epilog = ''
 parser = OptionParser(usage=usage,description=description,epilog=epilog)#,formatter=MultiLineHelpFormatter())
-parser.add_option('--upstream-window',dest='upst_win',type='int',default=100000,help='window width in base pairs to consider promoter region [default: %default]')
-parser.add_option('--downstream-window',dest='dnst_win',type='int',default=0,help='window width in base pairs to consider downstream region [default: %default]')
-parser.add_option('--tss',dest='tss',action='store_true',default=False, help='calculate downstream window from transcription start site instead of transcription end site')
+parser.add_option('--upstream-window',dest='upst_win',type='int',default= 2000,help='window width in base pairs to consider promoter region [default: %default]')
```

Comment:
@Mkebede correct me if I'm wrong, but I believe we added these new defaults to bring map_peaks_to_known_genes into agreement with the default way we run the script. When called from garnet.py, it selects a windowsize of 2000 and turns on the TSS flag. So the default behavior of Omics Integrator should not change with these changes (but it does change the default behavior of map_peaks_to_known_genes when run by itself, which will mostly only happen during our testing)